.github/scripts/check_boilerplate.sh

```diff
@@ -0,0 +1,238 @@
+#!/bin/bash
+
```

Comment:
Is there a public domain header?

Reply:
I added a mention about it in the NOTICE :)

I was about to add a mention about examples/ but since those were already there, I can't change their license. I think those should be public domain or any kind of CC or whatever so people copying those and using as a base of something wouldn't be breaching any licenses :)